src/elements/_OcSidebarNavItem.vue

```diff
@@ -38,11 +34,6 @@ export default {
       required: false,
       default: "",
     },
-    text: {
```

Comment:
please research on vuejs v-text - there was some special handling/support .... the text can be entered via prop or in a slot ..... if I recall correctly